apps/livreGUI/tfEditor/TransferFunctionEditor.cpp

```diff
@@ -275,24 +276,42 @@ void TransferFunctionEditor::_load()
 void TransferFunctionEditor::_save()
 {
     QString filename = QFileDialog::getSaveFileName( this, "Save transfer function",
-                                                           QString(),
-                                                           TF_FILE_FILTER );
+                                                     QString(),
+                                                     TF_FILE_FILTER );
     if( filename.isEmpty( ))
         return;
 
     if( !filename.endsWith( ".tf" ))
         filename.append( ".tf" );
 
+    // save .tf transfer function file (loadable via GUI)
     QFile file( filename );
     file.open( QIODevice::WriteOnly );
-    QDataStream out( &file );
-    out.setVersion( QDataStream::Qt_5_0 );
-
-    out << TF_FILE_HEADER << TF_FILE_VERSION;
-    out << _redWidget->getPoints()
-        << _greenWidget->getPoints()
-        << _blueWidget->getPoints()
-        << _alphaWidget->getPoints();
+    QDataStream outTf( &file );
+    outTf.setVersion( QDataStream::Qt_5_0 );
+
+    outTf << TF_FILE_HEADER << TF_FILE_VERSION;
+    outTf << _redWidget->getPoints()
+          << _greenWidget->getPoints()
+          << _blueWidget->getPoints()
+          << _alphaWidget->getPoints();
+
```

Comment:
Livre can only load .1dt, GUI only .tf. Unless we unify both tools to one format for load/save, we need two formats. Selecting which format to save from the GUI is probably cleaner. The GUI could also be used stand-alone w/o Livre, so saving the TF from there should still be possible imo.